src/apiCalls.js

```diff
@@ -20,8 +20,60 @@ export const getAllVendors = () => {
         });
 }
 
+export const getAllMarketVendors = () => {
+    return fetch(process.env.REACT_APP_BACKEND_URL + '/graphql?query=query{market_vendors{id market{id name}vendor{id name}}}')
+        .then(response => {
+            if (!response.ok) {
+                throw Error('Error fetching market vendor links');
+            }
+            return response.json();
+        });
+}
+
+export const getMarketsByVendor = (id) => {
+    return fetch(process.env.REACT_APP_BACKEND_URL + `/graphql?query=query{vendor(id: ${id}){markets{id name}}}`)
+        .then(response => {
+            if (!response.ok) {
+                throw Error('Error fetching markets');
+            }
+            return response.json();
+        });
+}
+
+export const deleteMarketVendorLink = (id) => {
```

Comment:
Great Job in getting this functionality for us to be able to delete connected Markets from a Vendors account.